doc/source/devel/howto_macros/macros_general.rst

```diff
@@ -613,7 +617,9 @@ parameters with different *flavors*:
     use :ref:`repeat parameters <sardana-macro-repeat-parameters>`.
     From Sardana 2.0 the repeat parameter values must be passed as lists of
     items. An item of a repeat parameter containing more than one member is a
-    list. In case when a macro defines only one repeat parameter
```

Comment:
After implementing changes agreed in the last #405 comments this is not true (execMacro with arguments as concrete types has not changed). Please undo this change if you agree.